File: 1. 23.08/python/3.py
def min(a,b):
    if a <= b:
        return a;
    else :
        return b;

num1 = int(input("Enter 1 number: "))
num2 = int(input("Enter 2 number: "))
num3 = int(input("Enter 3 number: "))
num4 = int(input("Enter 4 number: "))

if num1+num2+num3+num4 == 0:
    print("Minimum is: ",min(num1,min(num2,min(num3,num4))));
else :
    print("Sum is not 0");

File: 1. 23.08/C++/3.cpp
#include <iostream>

int find_min(int num1, int num2)
{
	return num1 <= num2 ? num1 : num2;
}

int main()
{
	int a,b,c,d;
	std::cout << "Enter 4 Nums:\n";
	std::cin >>a>>b>>c>>d;
	if (a+b+c+d == 0)
	{
		std::cout <<"Min Num is: " << find_min(find_min(find_min(a,b),c),d) << std::endl;
	}
	std::cout<<"Sum is not 0" << std::endl;
	return 0;
}
